elastic/eui
# Remove `sass-vars-to-js-loader` dependency

## Summary

This PR is part of a higher-level goal of removing our `node-sass` dependency (see https://github.com/elastic/eui/issues/6442).

This PR handles 2 remaining instances in our `src-docs/` calling `sass-vars-to-js-loader` by either importing the JSON variables directly or using the `useJsonVars` util added in #5227.

The biggest actual change (which I will highlight several areas of uncertainty) was `sass-vars-to-js-loader` being called in our exported Elastic Charts themes. As this affects actual source code vs docs only, it is by far the highest risk (although in theory it should work as before).

[I **strongly** recommend code-reviewing by commit.](https://github.com/elastic/eui/pull/6444/commits)

## QA

- Go to https://eui.elastic.co/pr_6444/#/theming/typography/values#font-weight?themeLanguage=sass
  - [x] Confirm the "I am proper bold" is still bold and has the same font-weight as production
  - [x] Confirm the "Variable fonts" range slider still looks and works as production does
- Go to https://eui.elastic.co/pr_6444/#/elastic-charts/creating-charts#theming-via-eui
  - [x] Confirm all charts on the page have the same color/appearance as production

### General checklist

~- [ ] A **[changelog](https://github.com/elastic/eui/blob/main/wiki/documentation-guidelines.md#changelog)** entry exists and is marked appropriately~ Open to thoughts, but since this in theory should not impact either end users or consumers, I skipped the changelog
